src/utils/execCommand.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import type Project from '../Project';
+import type Package from '../Package';
+import * as processes from './processes';
+import * as options from './options';
+
+export default async function execCommand(
+  project: Project,
+  pkg: Package,
+  command: string,
+  commandArgs: options.Args
+) {
+  let PATH_PARTS = [];
+
+  // We don't want to add the project package to the PATH twice
+  if (!pkg.isSamePackage(project.pkg)) {
+    PATH_PARTS.push(pkg.nodeModulesBin);
+  }
+
+  PATH_PARTS.push(project.pkg.nodeModulesBin);
+
+  if (process.env.PATH) {
+    PATH_PARTS.push(process.env.PATH);
+  }
+
+  let PATH = PATH_PARTS.join(':');
+
+  return await processes.spawn(command, commandArgs, {
+    cwd: pkg.dir,
+    tty: false,
```

Comment:
Do we want this to always be false? Maybe the commands should pass this in so that Project and Workspace (and maybe even workspaces if you filter it to contain one ws?) would get tty

Reply:
Fine for now either way

Reply:
I need to find a way to better support multiple processes logging at the same time.

I left it like this to revisit